src/gsi/setupaod.f90

```diff
@@ -827,6 +827,7 @@ subroutine contents_netcdf_diag_
       ! subroutine to write contents to netcdf diag files
       ! original: pagowski
       ! modified: 2019-03-21 - martin - cleaned up to fit GSI coding norms
+      use screen_to_ncdiag
```

Comment:
As noted before, we could add `screen_to_ncdiag` to module `ncdiag`.   If this were done, replace this line with `use ` from the appropriate module in `ncdiag`.

Reply:
Addressed.